dpsim/src/DAESolver.cpp

```diff
@@ -153,7 +154,7 @@ void DAESolver::initialize(Real t0) {
   //	}
   std::cout << "Call IDAInit" << std::endl;
 
-  ret = IDAInit(mem, &DAESolver::residualFunctionWrapper, t0, state, dstate_dt);
+  ret = IDAInit(mem, &DAESolver::residualFunctionWrapper, mt0, state, dstate_dt);
   //	if (check_flag(&ret, "IDAInit", 1)) {
   //		throw CPS::Exception();
   //	}
```

Comment:
Maybe we should remove this dead code as well?

Reply:
Do you mean the actual file or the commented lines below?